robot/2023/src/main/java/frc/robot/Path.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot;
+
+public class Path extends Robot
+{
+    //these may need to be floats depending on the encoder, I'll just wait and see though
+    double target;
```

Comment:
Note:  At some point, we may need a leftTarget and rightTarget.

Reply:
Added left and right targets